desktop-widgets/groupedlineedit.cpp

```diff
@@ -32,6 +32,9 @@
 #include <QScrollBar>
 #include <QTextBlock>
 #include <QPainter>
+#if QT_VERSION >= QT_VERSION_CHECK(5, 15, 0)
+#include <QPainterPath>
+#endif 
```

Comment:
you didn't need the version check here - and I believe this is already in master by now, anyway

Reply:
I will just drop this hunk in the merge as we already have this in master